packages/lab/stories/tooltip.doc.stories.mdx

```diff
@@ -23,20 +26,145 @@ A Tooltip provides guidance to the user when something requires more explanation
   <Story id="lab-tooltip--default" />
 </Canvas>
 
-# API
+## Features
+
+### Status
+
+Use the `status` prop to choose between `error`, `warning`, `success` and `info`.
+
+<Canvas>
+  <Story name="Tooltip Status" decorators={[withFlexGap]}>
+    <Tooltip content="I am a tooltip" status="info">
```

Comment:
in the examples where you have a status, there is a space missing between the status icon and the content. Idk if we want that to be added as part of the component or just a simple space in the example's content

Reply:
Looks fine locally. Seems it's a css specificity issue.

Reply:
Done